script/QA/QA_16.txt

```diff
@@ -81,72 +84,80 @@ just because it looks fun and interesting, okay?
 [sha:d414f7b419bbedd467e6ba10e286588472deb6be]{
 -- Page 82, Offset 43468.
 -- ……あの。
-...Uhh.
+Um...
 }
 [sha:ff8857dddf399ab9e4a59c48b443f3fdaa427b13]{
 -- Page 82, Offset 43469.
 -- さっきから静かですけど、なにやってるんですか？
-You've been kind of quiet for a while. What are you doing?
+ You've been kind of quiet for a while now.%{n}
+What are you doing?
 }
 [sha:dfbd8ce4204f9f6075a71fad0fd80f3b6e448897]{
 -- Page 83, Offset 43470.
 -- ん、アンケート書いてるとこ。
-Mmm, I'm filling out a questionnaire.
+// Not really a 'questionnaire' in context here, and this fits the context of patch.
+Mmm, I'm filing a bug report.%{n}
 }
 [sha:300aa2cd43f74670549046f92bf2cfac15764b8a]{
 -- Page 83, Offset 43471.
 -- 不具合の報告。
-Reporting a glitch.
+There's been a glitch.
 }
 [sha:1f39b2b47435a0fd7960fed7a634744db027dfba]{
 -- Page 84, Offset 43472.
 -- ほら、本編でヒロインが
-See, it's been two whole days since
+See, it's been two whole days since%{n}
 }
 [sha:fee7a410b2a2a220ef9d71ef93c01dff2e4932de]{
 -- Page 84, Offset 43473.
 -- でなくなってからまるまる２日っしょ？
-the heroine disappeared from the main story, right?
+the heroine disappeared from the main%{n}
+story, right?
 }
 [sha:59bb0d51caf6881a4e15c29919087d3f98843b06]{
 -- Page 85, Offset 43474.
 -- こんな状態が続くとメガネ君のモチベにも
-If the situation continues like this, it will affect
+If things carry on like this, it'll affect%{n}
 }
 [sha:2fb172e76fb8ca1ba4ec68eeda64d8aed8ec81c7]{
 -- Page 85, Offset 43475.
 -- 影響するしさー。早くパッチあててもらわないとねー。
-Glasses-kun's motivation. I need to get this patched quick.
+Glasses-kun's motivation. I need to get this patched stat.
```

Comment:
Glasses-kun... is this a consistent pattern. Feels like we need something better for this nickname at least.